Adyen/settings.py

```diff
@@ -2,5 +2,11 @@
 
 BASE_PAL_URL = "https://pal-{}.adyen.com/pal/servlet"
 BASE_HPP_URL = "https://{}.adyen.com/hpp"
-API_VERSION = "v30"
+ENDPOINT_LIVE_SUFFIX = "-pal-live.adyenpayments.com"
```

Comment:
is this being used ?

Reply:
correct, it is not used so it can be removed